sky/data/data_utils.py

```diff
@@ -225,3 +321,123 @@ def run_upload_cli(command: str, access_denied_message: str, bucket_name: str):
                 raise exceptions.StorageUploadError(
                     f'Upload to bucket failed for store {bucket_name}. '
                     'Please check the logs.')
+
+
+def store_rclone_config(bucket_name: str,
+                        region: str,
+                        apply_changes: bool = True):
+    """creates a configuration files for rclone - used for
+    bucket syncing and mounting """
+    # pylint: disable=import-outside-toplevel
+    import textwrap
+
+    rclone_file = os.path.expanduser('~/.config/rclone/rclone.conf')
```

Comment:
I see several other places that are using `'~/.config/rclone/rclone.conf'`. So lets create `RCLONE_CONFIG_PATH = '~/.config/rclone/rclone.conf`' in storage.py and use that.

Reply:
Agreed. The path will reside only in `data_utils.RCLONE_CONFIG_PATH`. `cloud_stores.py` and `storage.py` will import it.